mac.sh

```diff
@@ -242,66 +194,23 @@ install_postgresql
 brew_install_or_upgrade 'redis'
 brew_launchctl_restart 'redis'
 brew_install_or_upgrade 'sops'
-brew_install_or_upgrade 'the_silver_searcher'
-brew_install_or_upgrade 'vim'
```

Comment:
did you consider adding smaller scripts for folks to opt-in to if they want some tools installed, like `vim` or `oh_my_zsh`?

Reply:
yeah I talked through this with @showell215 @gummoe @jsonunger - but we all agreed most of that would come based on what someone personally likes or sees in a team buddies setup.

I would like to make the install a little more interactive or choice-based. I think when that comes, we could add in things like node/ruby version manager choice, vim / nvim / vscode, oh my zsh, various plugins, etc.

Reply:
Got it, yeah interaction-based would be nice. Sounds good.

I wonder, though, if something like core language version managers should be standardized so it's easier for folks to communicate when issues arise. Just my take, not massively opinionated on that.

Reply:
I love the idea of an interactive installer. When it comes to things like version managers, I originally felt similarly that it should be standardized, but the reality is that so long as you use the right version in whatever repo you're in, I don't see why we need to prescribe a specific manager "globally".